spec/i18n/placeholders_spec.rb

```diff
@@ -0,0 +1,45 @@
+describe "Placeholders in strings" do
+  it "translations preserve placeholders in strings" do
+    dirs = [Rails.root.join('locale')]
+
+    Rails::Engine.subclasses.each do |engine|
+      dir = engine.root.join('locale')
+      dirs << dir if dir.exist?
+    end
```

Comment:
@Fryguy actually, I implemented something like what you're suggesting here already, but
as a part of the ui split work. Over there, `Vmdb::Gettext::Domains.paths` would give you
array of locale paths of all registered plugins / engines. Let's say this is a temporary thing :smile: